packages/template-block/src/hooks/useTemplateBlockData.ts

```diff
@@ -206,9 +206,12 @@ const getPreviewClasses = (
     textPositioning: TextPositioningType,
     textRatio: TextRatioType
 ): string => {
-    return hasPreview && [TextPositioningType.Right, TextPositioningType.Left].includes(textPositioning)
-        ? textRatioToInverseFlexBasis[textRatio]
-        : '';
+    const classes =
+        hasPreview && [TextPositioningType.Right, TextPositioningType.Left].includes(textPositioning)
+            ? textRatioToInverseFlexBasis[textRatio]
+            : '';
+
+    return `${classes} tw-w-full`;
```

Comment:
nit, otherwise looks good

```suggestion
    const flexBasis =
        hasPreview && [TextPositioningType.Right, TextPositioningType.Left].includes(textPositioning)
            ? textRatioToInverseFlexBasis[textRatio]
            : '';

    return `${flexBasis} tw-w-full`;
```